draft-yasskin-webpackage-use-cases.md

```diff
@@ -0,0 +1,365 @@
+---
+coding: utf-8
+
+title: Use Cases and Requirements for Web Packages
+docname: draft-yasskin-webpackage-use-cases-latest
+category: info
+
+ipr: trust200902
+area: gen
+workgroup: dispatch
+keyword: Internet-Draft
+
+stand_alone: yes
+pi: [comments, sortrefs, strict, symrefs, toc]
+
+author:
+ -
+    name: Jeffrey Yasskin
+    organization: Google
+    email: jyasskin@chromium.org
+
+informative:
+  ServiceWorkers: W3C.WD-service-workers-1-20161011
+
+--- abstract
+
+This document lists use cases for signing and/or bundling collections
+of web pages, and extracts a set of requirements from them.
+
+--- middle
+
+# Introduction
+
+People would like to use content offline and in other situations where
+there isn’t a direct connection to the server where the content
+originates. However, it's difficult to distribute and verify the
+authenticity of applications and content without a connection to the
+network. The W3C has addressed running applications offline with
+Service Workers ({{ServiceWorkers}}), but not the problem of
+distribution.
+
+
+# Use cases
+
+These use cases are in rough descending priority order. If use cases
+have conflicting requirements, the design should enable more important
+use cases.
```

Comment:
Is there some line in the list of priorities that we won't cross?  Is Http/2 push essential to this document? is offline browsing?